client_example_test.go

```diff
@@ -342,11 +342,13 @@ func ExampleClient_Stream() {
 
 	// create a stream
 	streamQuery, _ := fauna.FQL(`StreamingSandbox.all().toStream()`, nil)
-	events, err := client.Stream(streamQuery)
+	events, err := client.StreamFromQuery(streamQuery, nil)
 	if err != nil {
 		log.Fatalf("failed to subscribe to the stream value: %s", err)
 	}
-	defer events.Close()
+	defer func() {
+		_ = events.Close()
+	}()
```

Comment:
I don't see the motivation for wrapping this.

Reply:
to silence linter warnings about unhandled errors

Reply:
mmk, i'd rather either let the linter warn us, or actually do something with the error (like log it). but dropping it just to silence the fact that it's unhandled feels wrong.